src/gameobjects/Button.js

```diff
@@ -470,12 +470,20 @@ Phaser.Button.prototype.setUpSound = function (sound, marker) {
 * @param {Phaser.Button} sprite - The Button that the event occurred on.
 * @param {Phaser.Pointer} pointer - The Pointer that activated the Button.
```

Comment:
`fastClick` should be [documented](http://usejsdoc.org/tags-param.html) here. Something along the lines of ` * @param {boolean} [fastClick=false] - Whether an event should be fired when...`